classes/Tab.php

```diff
@@ -684,4 +707,50 @@ public static function getTabModulesList($idTab)
 
         return $modulesList;
     }
+
+    /**
+     * Get tab position by it's class name
+     *
+     * @param $className
+     * @return false|null|string
+     */
+    public static function getPositionByClassName($className)
+    {
+        return Db::getInstance()->getValue('SELECT position FROM '._DB_PREFIX_.'tab WHERE class_name = "'. pSQL($className) .'"');
+    }
+
+    /**
+     * Get position before a tab by class name
+     * @param $className
+     * @return bool|int
+     */
+    public static function getPositionBeforeClassName($className)
```

Comment:
I'm wondering ... If you want to set a tab before another one, don't you need to set the same position instead?

Reply:
You're right.  Since the tabs will be shifting to make room 'before' should be the same position.  I have corrected the method.